docker/entrypoint.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Inits node configuration and runs the node.
+# e -> exit immediately, u -> treat unset variables as errors and immediately, o -> sets the exit code to the rightmost command 
+set -euo pipefail
+
+# within the container, $HOME=/home/cheqd
+CHEQD_ROOT_DIR="$HOME/.cheqdnode"
+
+# Init node config directory
+if [ ! -d "${CHEQD_ROOT_DIR}/config" ]
+then
+    echo "Node config not found. Initializing."
+    cheqd-noded init "moniker-placeholder"
```

Comment:
Just to confirm, is the value of `CHEQD_NODED_MONIKER` in `container.env` used as the actual moniker, if it has been set?

Reply:
Yes